core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2992,6 +2992,11 @@ class KafkaApis(val requestChannel: RequestChannel,
     val preprocessingResponses = configManager.preprocess(original.data(),
       (rType, rName) => authHelper.authorize(request.context, ALTER_CONFIGS, rType, rName))
     val remaining = ConfigAdminManager.copyWithoutPreprocessed(original.data(), preprocessingResponses)
+    val isKRaftController = metadataSupport match {
```

Comment:
Could you please add comments for this check? It seems to me race condition always needs good docs.